### DISCUSSION TITLE
[Unified Alerting] Enable Search by Rule Names

### DISCUSSION THREAD

**Discussion by tonychoe** [grafana/grafana#56065] [Alerting]
It would be great if we can search recording/alerting rules by their name. Currently, only way to search them is by labels and that's sometimes not enough. 

<img width="1462" alt="Screen Shot 2022-09-29 at 7 00 18 PM" src="https://user-images.githubusercontent.com/12507518/193174009-abfed869-e43a-46e7-9cdd-c56c3ce7f01e.png">

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
